src/apps/lua_generic/test/lua_generic_test.cpp

```diff
@@ -318,6 +319,7 @@ TEST_CASE("simple bank")
   Store::Tx gen_tx;
   GenesisGenerator gen(network, gen_tx);
   gen.init_values();
+  gen.create_service({});
```

Comment:
Can that be rolled into init_values()?

Reply:
We should do that. However, the whole `GenesisGenerator` class needs splitting and I didn't want to start combining some functions before doing that. Ideally, there will eventually be a `init_service(cert)` that initialises the state of the service.